pyopensprinkler/station.py

```diff
@@ -199,6 +199,11 @@ def start_time(self):
         """Retrieve start time"""
         return self._controller._timestamp_to_utc(self._get_status_variable(2))
 
+    @property
+    def end_time(self):
+        """Retrieve end time"""
+        return self.start_time + self.seconds_remaining
```

Comment:
Should this use current time rather than `start_time` ?

Reply:
no since its relies on `refresh()` to grab data so the seconds remaining is not based on current time

Reply:
But its not based on start time either so this is wrong lol

Reply:
ok added seconds_remaining to last refresh

Reply:
Can I test my understanding on this one.

I had assumed that `start_time` was static based on the station schedule and `seconds_remaining` was a reducing value that ultimately goes to zero. So the logic above appears to have `end_time` approaching `start_time`. But I may be misunderstanding so appreciate any clarification.

I agree that you cant simply add `seconds_remaining` to the current HA time as it could introduce an error up to the refresh period. I think it makes sense to use the UTC equivalent of `devt` as the base time for adding `seconds_remaining` as they are in sync in the `refresh()` response.

Reply:
yea my last one was wrong, I have changed it now

Reply:
hmm good point about `devt` maybe I should use that

Reply:
Yeah, using `devt` should ensure `end_time` doesn't bounce.

Reply:
Ok switched again to `devt`, kept the last refresh time as a property as well even though they are most likely equal